jung-algorithms/src/main/java/edu/uci/ics/jung/layout/algorithms/AbstractIterativeLayoutAlgorithm.java

```diff
@@ -24,6 +25,12 @@
   protected boolean shouldPreRelax = true;
   protected int preRelaxDurationMs = 500; // how long should the prerelax phase last?
 
+  protected Random random = new Random();
+
+  public void setRandomSeed(long randomSeed) {
```

Comment:
Sounds good to me. We nearly always call the default CTOR now, and set nothing. I think some setters should still be public in case we ever want to change a setting on an existing layout algorithm without rebuilding it.

Reply:
Setters that make sense to set after layout construction should still be public, agreed.  Anything that only makes sense on initialization will be rolled into the builder.